dtm0/service.c

```diff
@@ -479,7 +497,7 @@ static bool process_clink_cb(struct m0_clink *clink)
 	 * trigger for handling HA messages in this case.
 	 */
 	if (evented_proc_state != M0_NC_TRANSIENT &&
-	    !ready_to_process_ha_msgs && !m0_dtm0_in_ut()) {
+	    !ready_to_process_ha_msgs) {
```

Comment:
Why m0_dtm0_in_ut() check is not needed anymore?

Reply:
It has been checked already at the start:

```c
	if (m0_dtm0_in_ut()) {
		M0_LOG(M0_DEBUG, "No HA handling in UT.");
		goto out;
	}
```

I don't think it is necessary to check this again. That's why I have removed it.

Reply:
Agreed.